FOS.Users.Api/FOS.Users.Api/FOS.Users.Api/Controllers/HomeController.cs

```diff
@@ -16,7 +16,7 @@ public class HomeController : FOSControllerBase
         /// Constructor For <see cref="HomeController"/>
         /// </summary>
         /// <param name="mediator"></param>
-        public HomeController(IMediator mediator, ILogger<UsersController> logger) : base(mediator, logger)
+        public HomeController(IMediator mediator, ILogger<AuthController> logger) : base(mediator, logger)
```

Comment:
It should be ILogger<HomeController> logger. We are logging for the HomeController